cmd/terramate/e2etests/list_test.go

```diff
@@ -195,6 +195,50 @@ terramate {
 	assertRunResult(t, cli.listChangedStacks(), want)
 }
 
+func TestListChangedIgnoreDeletedNonStackDirectoryInGitDiff(t *testing.T) {
```

Comment:
hmm wondering on adding GitDiff on the test names if it is a good idea, it is clear enough IMHO to just say "TestListChangedIgnoreDeletedNonStackDirectory".  

Unrelated to adding GitDiff or not, the name of the test is misleading, looking at the code the scenario is:

- Create stack
- Push on main
- Remove stack
- Create branch, check that it is not detected as changed

Wouldn't it be: "TestListChangedIgnoreDeletedStackDir" ? 

By "non-stack" dir I was expecting the test to delete some other random directory inside the project that was not a stack and checking that it doesn't cause any changes to be listed, not deleting an actual stack.

Reply:
Now the names are:

`TestListChangedIgnoreDeletedStackDirectory`: ListChanged() ignore deleted stack directory.

`TestListChangedIgnoreDeletedNonStackDirectory`: ListChanged() ignore deleted non-stack directory (ie: ignores a directory not part of a stack).

`TestListChangedDontIgnoreStackDeletedFiles`: ListChanged() do not ignore stack deleted files.

`TestListChangedDontIgnoreStackDeletedDirectories`: ListChanged() do not ignore stack deleted directories.

Are they ok?